oracle/blockeligibilityvalidator.go

```diff
@@ -42,6 +42,9 @@ func (v BlockEligibilityValidator) BlockEligible(block *types.BlockHeader) (bool
 			return false, err
 		}
 		activeSetSize = atx.ActiveSetSize
+		if atx.PubLayerIdx.GetEpoch(v.layersPerEpoch).IsGenesis() && activeSetSize < v.committeeSize {
```

Comment:
this should not happen

Reply:
of course it should. We publish `activeSetSize = 0` in genesis. We can actually skip the second part of the condition (it was added to allow ATXs in previous epoch even in genesis, but not relevant with 1 epoch genesis).

Reply:
btw, we could change this and publish the hard-coded active set size.